backend/crystal_library/src/graphql/crystal_wells_res.rs

```diff
@@ -59,6 +59,23 @@ impl CrystalWellsQuery {
             .one(db)
             .await?)
     }
+
+    /// Reference resolver for crystal wells in crystal library graphql subgraph
+    #[graphql(entity)]
+    async fn find_crystal_wells_by_id(
```

Comment:
Is there a way we could deduplicate this and the `crystal_well` query? Can we have a `#[graphql(entity)]` that is also exposed in the graph as a regular Query?